src/wallet/asyncrpcoperation_sendmany.cpp

```diff
@@ -251,6 +249,9 @@ bool AsyncRPCOperation_sendmany::main_impl() {
     for (SendManyInputJSOP & t : z_inputs_) {
         z_inputs_total += std::get<2>(t);
     }
+    for (auto t : z_sapling_inputs_) {
```

Comment:
We don't allow sprout and sapling notes to be consumed in the same tx.  So we could an assert here to check the condition that if z_inputs_ size is > 0, then z_sapling_inputs must be 0, and vice versa.